iot-hub/Samples/device/ConventionBasedOperations/readme.md

```diff
@@ -532,6 +540,29 @@ await _deviceClient.SubscribeToWritablePropertiesEventAsync(
     cancellationToken);
 ```
 
+## Notes
+
+The following points describe the behavior of the client when both convention-aware and non-convention-aware callbacks are subscribed to at the same time.
+
+### Commands
+
+Commands subscription is implemented over the default handler. This gives rise to the following scenarios:
+1. You've set both `SubscribeToCommandsAsync` and `SetMethodDefaultHandlerAsync`.
+    
+    Since both of these APIs set the default handler, the handler that is set the last will overwrite any previously set handler and that is the one which would always get triggered.
+
+2. You've set `SubscribeToCommandsAsync` and a method-specific `SetMethodHandlerAsync`. 
+    
+    This will set the callback supplied to `SubscribeToCommandsAsync` to the default handler while the callback supplied to `SetMethodDefaultHandlerAsync` would be set for the method-specific handler. The order in which these callbacks are set doesn't matter in this case.
+    - If you invoke the method (or command, in convention-aware terms) that was set via `SetMethodDefaultHandlerAsync`, `SetMethodDefaultHandlerAsync` callback would be triggered.
+    - Any other method (or command, in convention-aware terms) would invoke the `SubscribeToCommandsAsync` callback.
+
+### Properties
+
+Both convention-aware property updates (`SubscribeToWritablePropertyUpdateRequestsAsync`) and non-conventiom-aware twin desired property updates (`SetDesiredPropertyUpdateCallbackAsync`) are implemented over the default handler, i.e. property updates are not received on a per-property specific handler.
+
+Since both of these APIs set the default handler, the order in which these callbacks are set is relevant. The handler that is set the last will overwrite any previously set handler and that is the one which would always get triggered.
```

Comment:
```suggestion
Since both of these APIs set the default handler, the order in which these callbacks are set is relevant. The handler that is set the last will override any previously set handler and that is the one which would always get triggered.
```

Reply:
Since the callback handler is essentially getting replaced or overwritten, I feel like "overwrite" conveys the desired message. What do you say?

Reply:
I don't feel strongly about it, just a suggestion, my reasoning is that if it's a text or a value it will be overwritten, if it's a method implementation, it's overridden :D

Reply:
I discussed this with drwill and he agrees with you. Resolving.